editor/editor_tests/osm_feature_matcher_test.cpp

```diff
@@ -415,3 +411,62 @@ UNIT_TEST(HouseBuildingMiss_test)
   auto const bestWay = osm::GetBestOsmWayOrRelation(osmResponse, geometry);
   TEST_EQUAL(bestWay.attribute("id").value(), string("345630019"), ());
 }
+
+string const kHouseWithSeveralEntrances = R"xxx("
+<osm version="0.6" generator="CGImap 0.6.0 (3589 thorn-03.openstreetmap.org)" copyright="OpenStreetMap and contributors" attribution="http://www.openstreetmap.org/copyright" license="http://opendatacommons.org/licenses/odbl/1-0/">
+  <node id="339283610" visible="true" version="6" changeset="33699414" timestamp="2015-08-31T09:53:02Z" user="Lazy Ranma" uid="914471" lat="55.8184397" lon="37.5700770"/>
```

Comment:
Could you please create a helper function that will take a list of lat-lons and return generated osm xml? This will be more readable and flexible.